### ISSUE TITLE
mergeOptions does not work on leftButtonColor and rightButtonColor

### ISSUE THREAD

**Issue by robrechtme** [wix/react-native-navigation#6663]
### Issue Description

Applying mergeOptions with a new `leftButtonColor` or `rightButtonColor` does not change the colors of the buttons. 
However, when the original button is passed in the `mergeOptions`, the color does change.

### Steps to Reproduce / Code Snippets / Screenshots

This is a simplified version which can help you reproducing the issue:
```js
const Screen = ({ componentId }) => {

  const changeButtonColor = () => {
    Navigation.mergeOptions(componentId, {
      topBar: {
        leftButtonColor: 'blue', // this should turn the button blue, but it doesn't
      },
    })
  }

  return (
    <View>
      <Button label="Change button color" onPress={changeButtonColor} />
    </View>
  );
}

Screen.options = {
  topBar: {
    leftButtons: [
      {
        id: 'LEFT',
        icon: require('../../img/clear.png'),
      },
    ],
    leftButtonColor: 'red', // <--
  },
}

export default Screen;
```
It is also easily testable in the ButtonsScreen of the playground. Make sure you remove the color prop on the individual buttons.

---
### Environment
* React Native Navigation version: 7.1.0
* React Native version: 0.63.2
* Platform(s) (iOS, Android, or both?): both
* Device info (Simulator/Device? OS version? Debug/Release?): Tested on Android 11 and iOS 14 (Simulator)

**Comment by robrechtme** [comment]
Still an issue.

**Comment by pribeh** [comment]
Experiencing the same.

**Comment by swabbass** [comment]
Thx for the report:] since those fields are going to be deprecated, I'm suggesting meanwhile to use [OptionsProcessor](https://wix.github.io/react-native-navigation/next/docs/style-theme#conditional-themes-with-options-processor), set default options for buttons example, this is more elegant.

For now, I'm working on a fix you can see draft #6882, till next major.
